slices/ovn-common.yaml

```diff
@@ -0,0 +1,25 @@
+package: ovn-common
+
+slices:
+  core:
```

Comment:
Again asking for info because I don't have much info on the package. What is the rationale behind putting these files in core?

Reply:
That's a good question - I'm not entirely clear on when a split should be made between bins and core - it might make more sense for this all to be in once slice as I'm not sure core is useful without bins.